apidoc/Titanium/Platform/Platform.yml

```diff
@@ -208,6 +208,8 @@ properties:
     summary: |
         Name of the platform. Returns `android` for the Android platform, `iPhone OS` for the iOS
         platform (iPhone, iPad, or iPod Touch), `windows` for the Windows platform, and `mobileweb` for the Mobile Web platform.
+        
+        Since 5.5.0.GA return 'iOS' for for the iOS platform (iPhone, iPad, or iPod Touch).
```

Comment:
I just noticed Ti.Platform.name return `iPhone OS` with Xcode 8 and iOS simulator 9.3, so it has to be.

```
On iOS 10+ return 'iOS' for for the iOS platform (iPhone, iPad, or iPod Touch).
```